site/rcuinfo.md

```diff
@@ -0,0 +1,46 @@
+## Specifying RCU connection information in the model
+
+During creation or updating a domain. There is new section RCUDbInfo in the model under domainInfo section for specifying RCU database connection information without using the command line options -rcu_database and -rcu_prefix.  This is created to support database where the connection string is more complicated and requires extra options.
+
+For example, in order to use Oracle Autonomous Transaction Processing Cloud Database for the JRF domain, specify the following information in the model:
+
+```yaml
+domainInfo:
+    RCUDbInfo:
+        useATP : 1
+        rcu_prefix : DEV
+        rcu_schema_password : xxxxx
+        rcu_admin_user : admin
+        tns.alias : dbatp_tp
+        javax.net.ssl.keyStorePassword : xxxx
+        javax.net.ssl.trustStorePassword : xxxx
+```           
+The database wallet can be included in the archive file under atpwallet/ structure
+
+atpwallet/Walletxyz.zip
+        
+Using createDomain tool with the -run_rcu flag will create the rcu schemas against the ATP database and configure the JRF domain
```

Comment:
Using createDomain tool with the -run_rcu flag will create the rcu schemas -> Using the Create Domain Tool with the `-run_rcu` flag, will create the RCU schemas